src/ucp/proto/proto_select.inl

```diff
@@ -29,9 +29,11 @@ KHASH_IMPL(ucp_proto_select_hash, khint64_t, ucp_proto_select_elem_t, 1,
 
 
 static UCS_F_ALWAYS_INLINE const ucp_proto_threshold_elem_t*
-ucp_proto_thresholds_search(const ucp_proto_threshold_elem_t *thresholds,
+ucp_proto_thresholds_search(const ucp_proto_select_elem_t *select_elem,
```

Comment:
minor: maybe pass `thresholds` to this func instead?

Reply:
in all places the function is used, it's taking thresholds from select_elem, so it's a small code simplification to just pass select_elem
rename the function to reflect the change

Reply:
ok